<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanDialog</class>
 <widget class="QDialog" name="BanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(245, 245, 245);
	border-radius: 20px;
	height: 200px;
	width: 300px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="descriptionLabel">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(65, 47, 50);
	padding-left: 10px;
	height: 30px;
}</string>
     </property>
     <property name="text">
      <string>Введите никнейм пользователя и выберите нужную опцию.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="unbanButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
     </property>
     <property name="text">
      <string>Разблокировать</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="banButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
     </property>
     <property name="text">
      <string>Заблокировать</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="usernameEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: rgb(245, 245, 245);
	height: 50px;
	color: rgb(60, 47, 50);
	margin-top: 5px;
	padding-left: 10px;
	border-bottom: 3px solid rgb(60, 47, 55);
	border-radius: 1px;
	font-size: 20px;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
